src/clib/pio_internal.h

```diff
@@ -152,6 +152,9 @@ extern "C" {
     int compute_counts(iosystem_desc_t *ios, io_desc_t *iodesc, int maplen,
                        const int *dest_ioproc, const PIO_Offset *dest_ioindex,
                        MPI_Comm mycomm);
+
+    /* Compute mapping for subset rearranger. */
+    int default_subset_partition(iosystem_desc_t *ios, io_desc_t *iodesc);    
```

Comment:
Thanks I have added that comment to the code. Keep them coming. Lots of ??? in pio_rearranger.c...